powerauth-webflow-authentication-mtoken/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/mtoken/controller/MobileAppApiController.java

```diff
@@ -153,8 +156,8 @@ protected AuthMethod getAuthMethodName() {
      * @throws PowerAuthAuthenticationException Thrown in case PowerAuth authentication fails.
      */
     private ObjectResponse<OperationListResponse> getOperationListImpl(PowerAuthApiAuthentication apiAuthentication) throws InvalidActivationException, PowerAuthAuthenticationException {
-        if (apiAuthentication != null && apiAuthentication.getUserId() != null) {
-            String activationId = apiAuthentication.getActivationId();
+        if (apiAuthentication != null && apiAuthentication.getUserId() != null && apiAuthentication.getActivationContext() != null) {
```

Comment:
This is a bit strange. If `apiAuthentication != null`, can it be that `apiAuthentication.getActivationContext() == null`? Do we do this check just because the activation ID is frequently accessed? I would place activation ID back in the authentication object...

Reply:
Right, we should check the activation ID instead, this is a mistake.